### PR TITLE
Added Meetup link to Contact page

### LINKED ISSUE #19: Add a link to Cincy WWCode meetup page

### PR DESCRIPTION
**Pull request by BayLadyCoder**
@sandiritter @sierraobryan @alexxmitchell  Please review. Thank you :)

## Description

- Added Meetup link to Contact page.
- Capitalized the first character of the links "twitter" and "facebook" to "Twitter" and "Facebook"
- Added `target="_blank"` and `rel="noreferrer"` to all the `<a>` tags, It opens a new tab when users click the links, so the user never have to leave the page.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation content changes
- [ ] Other (please describe)

## List any open issues this change will close
closes #19 

## Describe how to test/validate these changes
See screenshots below.

### Before
Before

### After
After

### CODE REVIEW

File: src/pages/contact.js [hunk 1]
```diff
@@ -20,8 +21,21 @@ const ContactPage = () => {
...
+        </a>
+      </p>
+      <p>
+        <a href={contactInfo.contact.links.facebook} target="_blank">
+          Facebook
+        </a>
+      </p>
+      <p>
+        <a href={contactInfo.contact.links.meetup} target="_blank">
+          Meetup
```
Comment (alexxmitchell) [inline src/pages/contact.js]: @BayLadyCoder Can you add `rel="noreferrer` to the links? There's a warning that comes up when running it:
Screen Shot 2020-11-01 at 8 35 23 AM

```suggestion
        <a href={contactInfo.contact.links.twitter} target="_blank" rel="noreferrer">
          Twitter
        </a>
      </p>
      <p>
        <a href={contactInfo.contact.links.facebook} target="_blank" rel="noreferrer">
          Facebook
        </a>
      </p>
      <p>
        <a href={contactInfo.contact.links.meetup} target="_blank" rel="noreferrer">
          Meetup
```

**Review by alexxmitchell — CHANGES_REQUESTED** [review]

File: src/pages/contact.js (hunk 1 above)
Comment (BayLadyCoder) [reply src/pages/contact.js]: @alexxmitchell Working on it. Thanks for pointing it out! :)

Comment (BayLadyCoder) [reply src/pages/contact.js]: @alexxmitchell Please review. I just pushed the changes (Added rel="noreferrer" to all the links).

**Review by alexxmitchell — APPROVED** [review]
LGTM 👍

**Pull request merged**